apiserver/facades/client/spaces/shims.go

```diff
@@ -53,7 +53,7 @@ func (s *stateShim) AllSpaces() ([]networkingcommon.BackingSpace, error) {
 }
 
 func (s *stateShim) Subnet(cidr string) (networkingcommon.BackingSubnet, error) {
```

Comment:
Given that pattern seems to be X(y) -> return instance of X where ID is Y, shouldn't this become `SubnetByCIDR` to avoid confusion?